.github/custom-tests.yaml

```
@@ -0,0 +1,32 @@
+
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+    - name: Run Custom Test Script
+      env:
+        BLOB_READ_WRITE_TOKEN: ${{ secrets.BLOB_READ_WRITE_TOKEN }}
+
+      run: |
+        python tests/custom_test_script.py
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)